dexs/filament/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { SimpleAdapter } from "../../adapters/types";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+import { httpGet } from "../../utils/fetchURL";
+
+const api = "https://orderbook.filament.finance/sei/api/v1/orderbook/tradeVolumeStats/BTC";
+
+const fetch = async () => {
+  const timestamp = getUniqStartOfTodayTimestamp();
+  const res = await httpGet(api);
+  const { allTimeVolume, volumeIn24Hours } = res;
+
+  return {
+    timestamp,
+    dailyVolume: volumeIn24Hours,
+    totalVolume: allTimeVolume,
+  };
+};
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    near: {
```

Comment:
why fees is  sei but dexs volume is `near` chain ?

Reply:
I was using chain from [[CHAIN.SEI] in both fees and volume.

I have changed to `sei` in case of volume now.